### PR TITLE
Do not return void expressions

### PR DESCRIPTION
**Pull request by Holzhaus**
Fixes a bunch of `-Wclazy-returning-void-expression` warnings, since
returning void expressions is confusing and may hide subtle bugs. See
https://github.com/KDE/clazy/blob/master/docs/checks/README-returning-void-expression.md
for details.

### CODE REVIEW

File: src/widget/paintable.cpp [hunk 1]
```diff
@@ -159,7 +159,7 @@ void Paintable::draw(const QRectF& targetRect, QPainter* pPainter,
                          math_min(sourceRect.height(), targetRect.height()));
         QRectF adjustedTarget(targetRect.topLeft(), fixedSize);
         QRectF adjustedSource(sourceRect.topLeft(), fixedSize);
-        return drawInternal(adjustedTarget, pPainter, adjustedSource);
+        drawInternal(adjustedTarget, pPainter, adjustedSource);
```
Comment (uklotzde) [inline src/widget/paintable.cpp:162]: We should use a switch/case here like we did in the `BaseTrackTableModel` ;)

Comment (Holzhaus) [reply src/widget/paintable.cpp:163]: Done.

**Review by uklotzde — APPROVED** [review]
LGTM. Waiting for GitHub CI.

**Comment by uklotzde** [comment]
No need to wait for Travis.

**Pull request merged**